File: interface/dart/server/yock_test.dart
import 'package:grpc/grpc.dart';

import '../yock.pb.dart';
import 'yock.dart';

@Call("SayHello")
Future<CallResponse> doSomething(ServiceCall call, CallRequest request) async {
  print(request);
  return new CallResponse(buf: "I'm Dart");
}

void main(List<String> args) {
  YockInterface(args).Start();
}

File: interface/python/server/yock_test.py
from server.yock import Call, Yocki
from yocki.yock_pb2 import CallRequest, CallResponse


@Call(fn="SayHello")
def sayHello(request: CallRequest):
    print("recv: ", request)
    return CallResponse(Buf="I'm Python")

Yocki.run()